app/build.gradle

```diff
@@ -14,37 +14,71 @@ android {
     }
 
     // Note the order of the flavors are important in generating the publishing artifacts
-    flavorDimensions "bundle", "api"
+    flavorDimensions "api"
     productFlavors {
-        basic {
-            dimension "bundle"
-        }
         minapi16 {
             dimension "api"
             minSdkVersion 16
+            ext {
+                artifactId = "${project.ext.bundleName}-${name}"
+                aarBaseName = name
+            }
         }
         minapi21 {
             dimension "api"
             minSdkVersion 21
+            ext {
+                artifactId = "${project.ext.bundleName}-${name}"
+                aarBaseName = name
+            }
         }
         androidTV {
             dimension "api"
             minSdkVersion 21
+            ext {
+                artifactId = "${project.ext.bundleName}-${name}"
+                aarBaseName = name
+            }
         }
         fireTV {
             dimension "api"
             minSdkVersion 21
+            ext {
+                artifactId = "${project.ext.bundleName}-${name}"
+                aarBaseName = name
+            }
         }
         unified {
             dimension "api"
             minSdkVersion 16
+            ext {
+                artifactId = 'unified'
+                aarBaseName = 'unifiedAndroid'
+            }
+        }
+        unified_ads {
+            dimension "api"
+            minSdkVersion 16
+            ext {
+                artifactId = 'unified-ads'
+                aarBaseName = "unifiedAndroid-ads"
+            }
+        }
+        unified_ads_ima {
+            dimension "api"
+            minSdkVersion 16
+            ext {
+                artifactId = 'unified-ads-ima'
+                aarBaseName = 'unifiedAndroid-ima'
+            }
         }
     }
 }
 
 dependencies {
-    basicImplementation 'com.google.code.gson:gson:2.8.2'
+    implementation 'com.google.code.gson:gson:2.8.2'
     unifiedImplementation 'com.google.android.gms:play-services-basement:17.6.0'
+    unified_ads_imaImplementation "com.google.ads.interactivemedia.v3:interactivemedia:3.25.1"
```

Comment:
To auto-import the ads module we need to also add it in the dependencies
`unified_ads_imaImplementation "com.theoplayer.theoplayer-sdk-android:unified-ads:${project.ext.sdkVersion}"`